backend/alexa_tutorial.md

```diff
@@ -0,0 +1,71 @@
+##Steps to building an alexa skill:
+
+###Prerequisites:
+In order to develop an Alexa skill you'll need access to 2 accounts.
+
+1. **aws.amazon.com**: An IAM user with access to the webservices@intrepid.io account
+2. **developer.amazon.com**:  webservices@intrepid.io for amazon developers, this login info is in LastPass.
```

Comment:
Yeah, I might leave this as is. I know that when the webteam was working there was a fair bit of confusion over which login to use.